README.md

```diff
@@ -4,110 +4,143 @@
-And the resource must pass **all** the rules to be allowed in the cluster.
+defined in the resources deployed in the cluster.
 
-The policy can either target Pods, or workload resources (Deployments, ReplicaSets,
-DaemonSets, ReplicationControllers, Jobs, CronJobs). Both have trade-offs:
+The policy can either target Pods, or workload resources (Deployments,
+ReplicaSets, DaemonSets, ReplicationControllers, Jobs, CronJobs). Both have
+trade-offs:
 
 - Policy targets Pods: Different kind of resources (be them native or CRDs) can
   create Pods. By having the policy target Pods, we guarantee that all the Pods
   are going to be compliant, even those created from CRDs. However, this could
   lead to confusion among users, as high level Kubernetes resources would be
-  successfully created, but they would stay in a non reconciled state.
-  Example: a Deployment creating a non-compliant Pod would be created, but it
-  would never have all its replicas running.
+  successfully created, but they would stay in a non reconciled state. Example: a
+  Deployment creating a non-compliant Pod would be created, but it would never
+  have all its replicas running.
 
-- Policy targets workload resources (e.g: Deployment): the policy inspect higher
-  order resource (e.g. Deployment): users will get immediate feedback about rejections.
-  However, non compliant pods created by another high level resource (be it native
-  to Kubernetes, or a CRD), may not get rejected.
+- Policy targets workload resources (e.g: Deployment): the policy inspect
+  higher order resource (e.g. Deployment): users will get immediate feedback
+  about rejections. However, non compliant pods created by another high level
+  resource (be it native to Kubernetes, or a CRD), may not get rejected.
 
 ## Settings
 
-Each rule defined in the policy settings is composed by a `reject` operator and a set
-of the environment variable used with the operator against the environment variables
-from the resources. The rules are evaluated in the order that they are defined.
-The resource is denied in the first failed evaluated rule. The following yaml is a settings example:
+> [!WARNING]  
+> If you are upgrading from version v1.x.x, please note the breaking changes
+> introduced in v2.x.x:
+>
+> Environment variable values are no longer considered.
+>
+> The policy now focuses solely on the name of the environment variable, not
+> its value.
+>
+> New settings syntax
+>
+> The settings no longer use a list of rules for validation. Instead, the
+> policy validates a single type of condition. Refer to the "Current Settings
+> Fields" section below for details on the new settings.
+
+The policy settings can include **one** of four fields:
+
+- `containsAnyOf`: enforces that the resource has at least one of the
+  `environmentVariables`.
+- `doesNotContainAnyOf`: enforces that the resource does not have any environment
```

Comment:
warning: this is the correct name to use from an English POV, but I've seen `doesNotContainsAnyOf` inside of the codebase (not the `s` of `Contain`). We should fix the codebase to use the correct English version